### PR TITLE
#1755 Disable PictureSlidesLab on empty presentation

### LINKED ISSUE #1755: Empty presentation throws NullReferenceException in PictureSlidesLab
#### Environment (e.g. version of Microsoft PowerPoint)
Windows 10, PowerPoint 2013
#### Steps to replicate
1. Delete all slides from presentation
2. Open picture slides lab.
3. Try to add a picture or apply existing picture to a slide. 
#### Expected behavior
Expected behaviour should be either error message informing the user that there are no slides to edit or to disable the button completely. I believe user should still be able to add a picture to the lab, but we can disable the button to apply the picture to the slide.
#### Actual behavior
Exception thrown.
```
System.NullReferenceException
  HResult=0x80004003
  Message=Object reference not set to an instance of an object.
  Source=PowerPointLabs
  StackTrace:
   at PowerPointLabs.PictureSlidesLab.Views.PictureSlidesLabWindow.ImageSelectionListBox_OnPreviewMouseDown(Object sender, MouseButtonEventArgs e) in C:\Users\Bryan\Desktop\PowerPointLabs\PowerPointLabs\PowerPointLabs\PictureSlidesLab\Views\PictureSlidesLabWindow.xaml.cs:line 688
```

### PR DESCRIPTION
**Pull request by ChesterSng**
Fixes #1755 

1. Disable PictureSlidesLab Button when there are no slides
2. Disable PictureSlidesLab Button and close existing PictureSlidesLab window when last slide is removed

**Outline of Solution**
This PR follows [PR 1715](https://github.com/PowerPointLabs/PowerPointLabs/pull/1715) closely to disable the PictureSlidesLab button when there are no slides in the presentation.

### CODE REVIEW

File: PowerPointLabs/PowerPointLabs/ActionFramework/PictureSlidesLab/PictureSlidesEnabledHandler.cs [hunk 1]
```diff
@@ -0,0 +1,16 @@
+﻿using PowerPointLabs.ActionFramework.Common.Attribute;
+using PowerPointLabs.ActionFramework.Common.Interface;
+using PowerPointLabs.TextCollection;
+using PowerPointLabs.TimerLab;
+
+namespace PowerPointLabs.ActionFramework.TimerLab
```
Comment (blewjy) [inline PowerPointLabs/PowerPointLabs/ActionFramework/PictureSlidesLab/PictureSlidesEnabledHandler.cs]: I'm guessing you took this from TimerLab... Probably should change the namespace? :P

File: PowerPointLabs/PowerPointLabs/PictureSlidesLab/PictureSlidesLab.cs [hunk 2]
```diff
@@ -0,0 +1,8 @@
+﻿namespace PowerPointLabs.PictureSlidesLab
+{
+    internal static class PictureSlidesLab
+    {
+#pragma warning disable 0618
```
Comment (blewjy) [inline PowerPointLabs/PowerPointLabs/PictureSlidesLab/PictureSlidesLab.cs]: Is this necessary for the code to work?

File: PowerPointLabs/PowerPointLabs/ActionFramework/PictureSlidesLab/PictureSlidesEnabledHandler.cs (hunk 1 above)
Comment (ChesterSng) [reply PowerPointLabs/PowerPointLabs/ActionFramework/PictureSlidesLab/PictureSlidesEnabledHandler.cs]: Changed!

File: PowerPointLabs/PowerPointLabs/PictureSlidesLab/PictureSlidesLab.cs (hunk 2 above)
Comment (ChesterSng) [reply PowerPointLabs/PowerPointLabs/PictureSlidesLab/PictureSlidesLab.cs]: Not really sure about this, removing this and running pptlabs does not seem to cause problems. However, in the TimerLab implementation of this, there was this line.

**Comment by blewjy** [comment]
Tested and working on both PPT2016 and PPT2010!

**Pull request merged**